packages/language-server/src/completion/completions.ts

```diff
@@ -461,7 +549,22 @@ function getDefaultValues(currentLine: string, lines: string[]): CompletionItem[
   }
 
   switch (fieldType) {
+    case 'BigInt':
     case 'Int':
+      if (datasourceProvider === 'cockroachdb') {
+        suggestions.push({
+          label: 'sequence()',
+          kind: CompletionItemKind.Function,
+          documentation:
+            'Create a sequence of integers in the underlying database and assign the incremented values to the ID values of the created records based on the sequence.',
+        })
+
+        if (fieldType === 'Int') {
+          // @default(autoincrement()) is only supported on BigInt fields for cockroachdb.
+          break
+        }
+      }
+
       suggestions.push({
         label: 'autoincrement()',
```

Comment:
I don't know if I am reading this correctly, but `Int @default(autoincrement())` is not valid on cockroachdb anymore (it has to be BigInt), and it looks like we offer it as a suggestion there.

Reply:
Tom yes, the logic is maybe not the most obvious, but it works like you described, see the tests at 
https://github.com/prisma/language-tools/pull/1148/files#diff-57989c19614e213ee18baa44c0b04488e590714de51ea5c3e1dbd34a6df1e09aR2140-R2169

autoincrement is only suggested for BigInt
<img width="696" alt="Screen Shot 2022-05-06 at 17 46 18" src="https://user-images.githubusercontent.com/1328733/167167440-eac6a0b1-94d4-46fa-8cc0-42f11a1e156b.png">

Reply:
Magic code that excludes `Int` is here:
https://github.com/prisma/language-tools/pull/1148/files#diff-8d75c7369dc815b1f54961c3730fab99d9fac36afc0bb9239f568e825d6f8706R554-R560
Also missed that on the first read.